hypertrace-gradle-publish-maven-central-plugin/README.md

```diff
@@ -56,5 +56,6 @@ plugins {
 
 #### Releasing repositories
 ```bash
+FIXME: This has changed, as has publish https://github.com/gradle-nexus/publish-plugin/wiki/Migration-from-gradle_nexus_staging-plugin---nexus_publish-plugin-duo#tasks---simple-case
```

Comment:
publish needs to be rewired, as well as updating consumers to use new task names.